<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_set_job_info</class>
 <widget class="QDialog" name="D_set_job_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="l_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>169</green>
         <blue>169</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>169</green>
         <blue>169</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>169</green>
         <blue>169</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>254</green>
         <blue>253</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(&quot;:/new_images/bg_full.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_close">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>62</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/button/btn_Exit-01a.png</normaloff>
     <activeoff>:/images/button/btn_Exit-01b.png</activeoff>:/images/button/btn_Exit-01a.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>49</width>
     <height>47</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="l_close">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>11</y>
     <width>37</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(&quot;:/new_images/b_close.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>공정등록</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_sub_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>아이템 선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_scan">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>98</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_next">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>510</y>
     <width>110</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 0, 0); 
Border-radius: 25px;
Border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>선택완료</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="select_process_room">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>305</width>
     <height>380</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="process10">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>288</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process7">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>194</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process2">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>5</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process3">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>5</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process1">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>5</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process6">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>99</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process5">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>100</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process8">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>194</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process9">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>194</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process4">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>99</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sunfont</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(112, 112, 112); 
Border-radius: 25px</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <zorder>process8</zorder>
   <zorder>process10</zorder>
   <zorder>process7</zorder>
   <zorder>process2</zorder>
   <zorder>process3</zorder>
   <zorder>process1</zorder>
   <zorder>process6</zorder>
   <zorder>process5</zorder>
   <zorder>process9</zorder>
   <zorder>process4</zorder>
  </widget>
  <widget class="QLabel" name="l_select_process_room">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>※ 공정번호 선택
( 좌측에서 공정을 선택한 후,
공정번호를 누르세요 )</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_confirm_message1">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>180</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  선택 하신 번호에 다음과 같이 저장된 공정이 있습니다.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_item_title">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>아이템 : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_process_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>공정 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_confirm_item">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>230</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>item</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_confirm_process">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>process</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_confirm_message2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  공정에 덮어 쓰시겠습니까?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="b_doNotReplace">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(75, 137, 220); 
Border-radius: 25px</string>
   </property>
   <property name="text">
    <string>아니오</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_replace">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(75, 137, 220); 
Border-radius: 25px</string>
   </property>
   <property name="text">
    <string>네</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_scan_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>111</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sunfont</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 0, 0); 
Border-radius: 25px;
Border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>재탐색</string>
   </property>
  </widget>
  <zorder>l_bg</zorder>
  <zorder>l_select_process_room</zorder>
  <zorder>l_close</zorder>
  <zorder>l_title</zorder>
  <zorder>b_close</zorder>
  <zorder>listWidget</zorder>
  <zorder>l_sub_title</zorder>
  <zorder>b_scan</zorder>
  <zorder>b_next</zorder>
  <zorder>select_process_room</zorder>
  <zorder>l_confirm_message1</zorder>
  <zorder>l_item_title</zorder>
  <zorder>l_process_title</zorder>
  <zorder>l_confirm_process</zorder>
  <zorder>l_confirm_message2</zorder>
  <zorder>l_confirm_item</zorder>
  <zorder>b_doNotReplace</zorder>
  <zorder>b_replace</zorder>
  <zorder>b_scan_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
